config.js

```diff
@@ -10,6 +10,8 @@ const CRON_FREQUENCY = process.env.EMAIL_CRON_PATTERN || '* * 1 * * *';
 const SECURE_CONNECTION = process.env.SECURE_CONNECTION == "true" || false;
 const EMAIL_ADDRESS = process.env.EMAIL_ADDRESS;
 const EMAIL_PASSWORD = process.env.EMAIL_PASSWORD;
+const LOG_ERRORS = process.env.LOG_ERRORS || false;
+const LOGS_GRAPH = process.env.LOGS_GRAPH || "http://mu.semte.ch/graphs/public";
```

Comment:
I find the name a little bit confusing, would something like `ERROR_LOGS_GRAPH,` work ? Otherwise it feels a bit like it's a "should I log something" variable. Personal opinion though, you might have a different one !

Reply:
Yeah I took the name from acmidm-login-service https://github.com/lblod/acmidm-login-service#configuration. Mayeb better to leave it like that just to stay consistent? I agree that ERROR_LOGS_GRAPH does sound more to the point... I leave the choice up to you

Reply:
After a little git blame, i'm seeing that I'm the one who poorly named it at first :see_no_evil: I'd say go rename it, not super bad not to have those 2 services consistent I guess

Reply:
ohh so you are the culprit haha. Sure should be quick fix